rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -373,29 +373,57 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
 
   RCLCPP_LIFECYCLE_PUBLIC
   const State &
+  trigger_transition(const Transition & transition, rcl_ret_t & error);
```

Comment:
Why is this function getting a `rcl_ret_t` argument while the other functions get a `rcl_lifecycle_ret_t`?